apps/pie-docs/src/components/modal/code/props.json

```diff
@@ -0,0 +1,190 @@
+{
+    "headings": ["Prop", "Options", "Description", "Default"],
+    "rows": [
+        [
+            "aria",
+            {
+                "type": "code",
+                "item": []
+            },
+            "An object representing the aria labels for the `close` & `back` buttons within the modal as well as the `isLoading` state labels (`aria-label` & `aria-busy`)."
+        ],
+        [
+            "hasBackButton",
+            {
+                "type": "code",
+                "item": ["false", "true"]
```

Comment:
Nit: Please can we have `true` before `false` for all the boolean props to match the other pages